sekoia_automation/aio/helpers/__init__.py

```diff
@@ -3,3 +3,24 @@
 
 NOTE!!!: each package inside requires additional libraries to be installed.
 """
+
+import asyncio
+import itertools
+from typing import AsyncGenerator, Any
+
+
+async def limit_concurrency(aws: list, limit: int) -> AsyncGenerator[Any, None]:
+    """
+    Limit the number of concurrent async tasks at the same time
+
+    :param list aws: The list of async tasks to execute
+    :param: int limit: The number of concurrent async tasks allows at the same time
+
+    """
+    aws = iter(aws)
```

Comment:
is it correct name? I mean `aws`

Reply:
Indeed, the name of the variable is not the better. Can be confused with AWS. However, it stands for `async workers`.

Reply:
I changed the name of the argument

Reply:
I changed the name of the argument